### PR TITLE
dequantization: int8 to fp32

### PR DESCRIPTION
**Pull request by abourramouss**

### CODE REVIEW

**Comment by abourramouss** [comment]
@hikettei it seems that the previous merge of RoPE didn't add my commits to the git history of the repo

**Comment by hikettei** [comment]
Yeah, it’s the one we discussed on Discord earlier. (I don’t quite understand the reason behind it. This is my first time encountering something like this.)

Is there any obstacle?

**Comment by abourramouss** [comment]
> Is there any obstacle?

No, straight forward, just merge the pr when ready and it should be enough to have a coherent git history

**Comment by abourramouss** [comment]
#274 currently working on this issue. Why f32 tensor blocks are not equal in pygguf and caten

File: Makefile [hunk 1]
```diff
@@ -10,6 +10,8 @@ help:
 .PHONY: install_extra
 install_extra: ## Install extra dependencies for testing
 	$(PIP) install numpy torch
+	git clone  https://github.com/99991/pygguf.git
```
Comment (hikettei) [inline Makefile:13]: i have forked 99991/pygguf: https://github.com/hikettei/pygguf
could you use this instead?

File: source/test-suite/caten.test-suite.asd [hunk 2]
```diff
@@ -23,7 +23,8 @@ Tests that are not related to the core functionality of Caten or are time-consum
                (:file "test-dynamic-shape")
                (:file "test-memory-planner")
                (:file "test-schedule-cache")
-               (:file "test-rope"))
+               (:file "test-rope")
+               (:file "external/test-gguf.lisp"))
   :perform
```
Comment (hikettei) [inline source/test-suite/caten.test-suite.asd:28]: no need to add .lisp

**Comment by hikettei** [comment]
Nice! will merge this as soon as ci becomes green

**Pull request merged**